.storybook/main.js

```diff
@@ -1,23 +1,27 @@
 const path = require("path");
 const glob = require("glob");
-
 const projectRoot = path.resolve(__dirname, "../");
 const ignoreTests = process.env.IGNORE_TESTS === "true";
 const isChromatic = !ignoreTests;
 const getStories = () =>
-  glob.sync(`${projectRoot}/src/**/*.stories.@(js|mdx|tsx)`, {
+  glob.sync(`${projectRoot}/src/**/*.@(mdx)`, {
+    ...(ignoreTests && {
+      ignore: `${projectRoot}/src/**/*.stories.@(js|mdx|tsx)`,
+    }),
+  });
+const getCSFStories = () =>
+  glob.sync(`${projectRoot}/src/**/*.stories.@(jsx|tsx)`, {
     ...(ignoreTests && {
       ignore: `${projectRoot}/src/**/*-test.stories.@(js|mdx|tsx)`,
     }),
   });
-
 module.exports = {
-  framework: "@storybook/react",
-  stories: (list) => [
-    ...list,
+  stories: [
     "./welcome-page/welcome.stories.js",
-    "../docs/*.stories.mdx",
+    "../docs/*.mdx",
+    "../docs/*.stories.tsx",
     ...getStories(),
```

Comment:
Yeah, as I say I think there's been an oversight here from the early days of the upgrade when upgrading storybook basically broke everything because of the incompatibility of stories.mdx. I may have put this in to try and help when I've been toying around trying to disable certain stories etc. This wasn't the solution I ended up using to disable stories during development, so yeah this can probably just be removed now